contrib/completion/bash/netctl

```diff
@@ -0,0 +1,463 @@
+#! /bin/bash
+
+GLOBAL_COMMANDS="\
```

Comment:
Looking more at the rest of the code/logic, it does seem that we are dependent on the top level objects, etc quite a bit. I am not sure if generating this code (or at least the parts of the code) is worth the gain, because then we can truly become independent of future changes.
So my suggestion is to not move to auto-generation of it but exploring this may benefit us in long run with highly sustainable code. ping @shaleman